Modules/_cursesmodule.c

```diff
@@ -3335,6 +3335,9 @@ PyInit__curses(void)
     SetDictInt("A_BLINK",               A_BLINK);
     SetDictInt("A_DIM",                 A_DIM);
     SetDictInt("A_BOLD",                A_BOLD);
+#ifdef A_ITALIC
+    SetDictInt("A_ITALIC",              A_ITALIC);
+#endif
```

Comment:
`A_ITALIC` is a ncurses extension, not X/Open standard. So I think it should be put after `A_VERTICAL` and prefixed with a line `/* ncurses extension */`.